tokenLists/cronos.tokenlist.json

```diff
@@ -4,7 +4,7 @@
         "kyberswap",
         "dmmexchange"
     ],
-    "timestamp": "2022-06-20T00:00:00+00:00",
+    "timestamp": "2022-07-27T14:30:00+07:00",
```

Comment:
What's this property for?

Reply:
It reflects here
<img width="763" alt="image" src="https://user-images.githubusercontent.com/23650671/181189004-3f850fc0-ef99-4ccd-9d42-f3f389523bc6.png">

![image](https://user-images.githubusercontent.com/23650671/181188961-61603677-5310-4ab2-9fda-4bb0104090fd.png)